lib/cocoapods/generator/copy_xcframework_script.rb

```diff
@@ -84,6 +84,10 @@ def script
   # Split archs on space so we can find a slice that has all the needed archs
   local target_archs=$(echo $ARCHS | tr " " "\\n")
 
+  # arm64e and arm7vs are not required architectures for a device slice
```

Comment:
while this may be true, aren't there certain devices that require these architectures? I don't think we want to ignore the checking completely but I might be misunderstanding what this is actually doing

Reply:
I think `armv7s` might be safe to drop, unsure about `arm64e` which is new for pointer authentication

Reply:
For one perspective on it, the App Store *at the moment* strips arm64e from submissions: https://developer.apple.com/documentation/security/preparing_your_app_to_work_with_pointer_authentication?language=objc
and `$ARCHS_STANDARD` is currently `arm64 armv7`.  
On the other hand, I realized after creating this that the ARCHS of the Pod Target is whats really being considered here.  That might make this entire concern semi-irrelevant (I need to look into the logic around ARCHS when the Pod target is created).

Reply:
CocoaPods does attempt to infer architectures from the user's project but its not perfect, and I don't think we have a way to determine the value of `ARCHS_STANDARD` when generating the pods project(s)

Reply:
Quick correction to a previous statement, `$ARCHS_STANDARD` is currently `arm64 armv7` for deployment targets < iOS 11.0.
With a deployment target of 11.0, `$ARCHS_STANDARD` it becomes just `arm64`.
So, `$ARCHS`, `$VALID_ARCHS`, and potentially deployment target (if $ARCHS_STANDARD is used) are all relevant to what the final result of `$ARCHS` will be when `copy_xcframework_script` sees it.

This is a factor in #9794, where the XCFramework could be installed manually (the App's deployment target is 13.4), but could not be installed by the Pod Target (deployment target of 10.0).

Reply:
ah sounds like https://github.com/CocoaPods/CocoaPods/issues/7314 we don't bump the deployment target of pods that support older versions in case that introduces warnings at build time.

it's still sounding to me like this xcframework should be updated to include the additional architectures OR the app should refine its architectures

Reply:
I have definitely concluded that this change to the xcframework script is incorrect.
For all intents and purposes, the response to an issue like  #9794 should be "Add the missing armv7 architecture, or set your Podspec's deployment target to 11.0 if you want to ship 64 bit only".

Before I put this to rest, my last question is should the `$VALID_ARCHS` of the Pod Target be set to reflect the contents of the XCFramework (from it's info.plist)?  Right now the App and the Pod target agree on `ARCHS` and `VALID_ARCHS`, but ultimately resolve to something different due to the difference in deployment target and its impact on `$ARCHS_STANDARD`.

Appreciate all the feedback you've provided on this!